packages/theme-chalk/src/tabs.scss

```diff
@@ -121,6 +121,11 @@
       outline: none;
     }
 
+    &:focus.is-active.is-focus:not(:active) {
```

Comment:
This works when you tab to the tabs, but once the selection changes the focus ring disappears.

I think all you need here is

```&:focus-visible {```

Reply:
@Giwayume Thanks for your help. After accessing the tabs with the keyboard tab, you can move the tabs using the arrow keyboard keys.

Reply:
@Giwayume sorry. Can you check again on #7665?